### PR TITLE
[cherry-pick] iterator lower bound and optimized ingesting sst

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**

### CODE REVIEW

File: src/coprocessor/select/select.rs [hunk 1]
```diff
@@ -152,7 +152,12 @@ impl<'a> SelectContext<'a> {
             } else {
                 range.get_start().to_vec()
             };
-            let upper_bound = if !self.core.desc_scan && !range.get_end().is_empty() {
+            let lower_bound = if !range.get_start().is_empty() {
```
Comment (hicqu) [inline src/coprocessor/select/select.rs]: Could we set both `lower bound` and `upper bound` like we do in function `init_with_range`?

Comment (zhangjinpeng1987) [reply src/coprocessor/select/select.rs]: Is is possible the start or end is empty?

**Comment by hicqu** [comment]
LGTM.

**Pull request merged**